### PR TITLE
Feature 1731 docs

### LINKED ISSUE #1731: Update the User's Guide for the 10.0.0 release.
## Describe the Task ##
Prior to the actual 10.0.0 release, we'll need to make some updates to the User's Guide. Please list the changes that need to be made here:
- Revise and rotate the User's Guide authorship list, as directed by @TaraJensen 
- Add 10.0.0 release notes (this will be done as part of the release process)
- Create a new 10.0.0 flowchart and drop it into the docs.
- Review/update section 1.4 "MET components" as needed.

Please note additional required changes here.

### Time Estimate ###
4 hours

### Sub-Issues ###
Consider breaking the task down into sub-issues.
No sub-issues needed.

### Relevant Deadlines ###
Prior to 10.0.0 release.

### Funding Source ###
2702691

### Assignee ###
- [x] Select **engineer(s)** or **no engineer** required: John HG and Julie
- [x] Select **scientist(s)** or **no scientist** required: John Opatz to review/edit changes

### Labels ###
- [x] Select **component(s)**
- [x] Select **priority**
- [x] Select **requestor(s)**

### Projects and Milestone ###
- [x] Review **projects** and select relevant **Repository** and **Organization** ones or add "alert:NEED PROJECT ASSIGNMENT" label
- [x] Select **milestone** to next major version milestone or "Future Versions"

## Define Related Issue(s) ##
Consider the impact to the other METplus components.
- [x] [METplus](https://github.com/dtcenter/METplus/issues/new/choose), [MET](https://github.com/dtcenter/MET/issues/new/choose), [METdatadb](https://github.com/dtcenter/METdatadb/issues/new/choose), [METviewer](https://github.com/dtcenter/METviewer/issues/new/choose), [METexpress](https://github.com/dtcenter/METexpress/issues/new/choose), [METcalcpy](https://github.com/dtcenter/METcalcpy/issues/new/choose), [METplotpy](https://github.com/dtcenter/METplotpy/issues/new/choose)
No impacts

## Task Checklist ##
See the [METplus Workflow](https://dtcenter.github.io/METplus/Contributors_Guide/github_workflow.html) for details.
- [ ] Complete the issue definition above, including the **Time Estimate** and **Funding Source**.
- [ ] Fork this repository or create a branch of **develop**.
Branch name: `feature_<Issue Number>_<Description>`
- [ ] Complete the development and test your changes.
- [ ] Add/update log messages for easier debugging.
- [ ] Add/update unit tests.
- [ ] Add/update documentation.
- [ ] Push local changes to GitHub.
- [ ] Submit a pull request to merge into **develop**.
Pull request: `feature <Issue Number> <Description>`
- [ ] Define the pull request metadata, as permissions allow.
Select: **Reviewer(s)**, **Project(s)**, **Milestone**, and **Linked issues**
- [ ] Iterate until the reviewer(s) accept and merge your changes.
- [ ] Delete your fork or branch.
- [ ] Close this issue.

### PR DESCRIPTION
**Pull request by j-opatz**
## Pull Request Testing ##

- [x] Describe testing already performed for these changes:</br>This branch was already merged, but additional documentation updates (tc_pairs, grid-diag) were added since the merge.

- [x] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>Quick language, spelling check should suffice.

- [x] Do these changes include sufficient documentation updates, ensuring that no errors or warnings exist in the build of the documentation? **[Yes or No]**</br>These are *only* documentation updates

- [x] Do these changes include sufficient testing updates? **[Yes or No]**

- [x] Will this PR result in changes to the test suite? **[Yes or No]**</br> No
If **yes**, describe the new output and/or changes to the existing output:</br>

- [x] Please complete this pull request review by **[Fill in date]**.</br>5/10, the major release

## Pull Request Checklist ##
See the [METplus Workflow](https://dtcenter.github.io/METplus/Contributors_Guide/github_workflow.html) for details.
- [x] Complete the PR definition above.
- [x] Ensure the PR title matches the feature or bugfix branch name.
- [x] Define the PR metadata, as permissions allow.
Select: **Reviewer(s)**, **Project(s)**, and **Milestone**
- [ ] After submitting the PR, select **Linked Issues** with the original issue number.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue and delete your feature or bugfix branch from GitHub.

### CODE REVIEW

File: met/docs/Users_Guide/tc-pairs.rst [hunk 1]
```diff
@@ -35,11 +35,11 @@ tc_pairs has required arguments and can accept several optional arguments.
 Required arguments for tc_pairs
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-1. The **-adeck source** argument indicates the adeck ATCF format data source containing tropical cyclone model forecast (output from tracker) data to be verified. It specifies the name of an ATCF format file or top-level directory containing ATCF format files ending in “.dat” to be processed. The **-adeck** or **-edeck** option must be used at least once.
+1. The **-adeck source** argument indicates the adeck TC-Pairs acceptable format data source containing tropical cyclone model forecast (output from tracker) data to be verified. Acceptable data formats are limited to the standard ATCF format and the one column modified ATCF file, generated from a genesis mode run for extratropical cyclones. It specifies the name of a TC-Pairs acceptable format file or top-level directory containing TC-Pairs acceptable format files ending in “.dat” to be processed. The **-adeck** or **-edeck** option must be used at least once.
```
Comment (JohnHalleyGotway) [inline met/docs/Users_Guide/tc-pairs.rst]:
```suggestion
1. The **-adeck source** argument indicates the adeck TC-Pairs acceptable format data source containing tropical cyclone model forecast (output from tracker) data to be verified. Acceptable data formats are limited to the standard ATCF format and the one column modified ATCF file, generated by running the tracker in genesis mode. It specifies the name of a TC-Pairs acceptable format file or top-level directory containing TC-Pairs acceptable format files ending in “.dat” to be processed. The **-adeck** or **-edeck** option must be used at least once.
```

Comment (JohnHalleyGotway) [reply met/docs/Users_Guide/tc-pairs.rst]: I'm pretty sure that tracker can be run in genesis mode for tropical or extra-tropical cyclones. I don't think it's only for extra-tropical.

File: met/docs/Users_Guide/tc-pairs.rst [hunk 2]
```diff
@@ -35,11 +35,11 @@ tc_pairs has required arguments and can accept several optional arguments.
...
 
-3. The **-bdeck source** argument indicates the ATCF format data source containing the tropical cyclone reference dataset to be used for verifying the adeck source. It specifies the name of an ATCF format file or top-level directory containing ATCF format files ending in “.dat” to be processed. This source is expected to be the NHC Best Track Analysis, but could also be any ATCF-formatted reference.
+3. The **-bdeck source** argument indicates the TC-Pairs acceptable format data source containing the tropical cyclone reference dataset to be used for verifying the adeck source. This source is typically the NHC Best Track Analysis, but could be any TC-Pairs acceptable formatted reference. The acceptable data formats for bdecks are the same as those for adecks. This arguement specifies the name of a TC-Pairs acceptable format file or top-level directory containing TC-Pairs acceptable format files ending in “.dat” to be processed.
```
Comment (JohnHalleyGotway) [inline met/docs/Users_Guide/tc-pairs.rst]:
```suggestion
3. The **-bdeck source** argument indicates the TC-Pairs acceptable format data source containing the tropical cyclone reference dataset to be used for verifying the adeck source. This source is typically the NHC Best Track Analysis, but could be any TC-Pairs acceptable formatted reference. The acceptable data formats for bdecks are the same as those for adecks. This argument specifies the name of a TC-Pairs acceptable format file or top-level directory containing TC-Pairs acceptable format files ending in “.dat” to be processed. 
```

**Review by JohnHalleyGotway — CHANGES_REQUESTED** [review]
Please see suggestions for minor edits.

File: met/docs/Users_Guide/tc-pairs.rst (hunk 1 above)
Comment (j-opatz) [reply met/docs/Users_Guide/tc-pairs.rst]: True; I want to convey that the one-column modified file was only created to better track extratropical cyclones, though. Would users have any need to run Guang Ping's extra ID string creator for tropical cyclones? If they would, then your edit is the better wording.

**Comment by j-opatz** [comment]
Comment changes have been approved

**Review by JohnHalleyGotway — APPROVED** [review]
Looks great, thanks John. I approve.

**Pull request merged**